app/src/main/res/font/metropolis.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <font-family xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+
+        <!-- Black -->
+        <font
+            android:font="@font/metropolis_black"
+            android:fontStyle="normal"
+            android:fontWeight="900"
+            app:fontStyle="normal"
+            app:fontWeight="900"
+            app:font="@font/metropolis_black"
+            />
```

Comment:
Nit: this isn't formatted according to our style guide, would you mind auto-formatting it?  (put your cursor in the file and press option + cmd + L in AS)